protocol/common/message.go

```diff
@@ -176,12 +178,16 @@ func (m *Message) PrepareServerURLs(port int) {
 	if m.ContentType == protobuf.ChatMessage_AUDIO {
 		m.AudioLocalURL = fmt.Sprintf("https://localhost:%d/messages/audio?messageId=%s", port, m.ID)
 	}
+	if m.ContentType == protobuf.ChatMessage_STICKER {
+		m.StickerLocalURL = fmt.Sprintf("https://localhost:%d/messages?sticker?messageId=%s", port, m.ID)
```

Comment:
better use hash directly here instead of messageId

Reply:
also `messages?sticker` should be `messages/sticker`

Reply:
URL also looks a bit funny, double `?`, or am I missing something?

Reply:
yeah, it's a typo :D